packages/client/src/service/LeaderBoardService.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosService } from './AxiosService';
+import { ApiEndpoint } from './types/api/enums/ApiEndpoint';
+import { IPlayer, IPlayerPayload } from './types/liderBoard/IPlayer';
+
+class LeaderBoardService extends AxiosService {
+  private readonly TEAM_NAME = 'bombers';
+  private readonly RATING_FIELD_NAME = 'score';
+
+  constructor() {
+    super();
+  }
+
+  public addPlayer(player: IPlayer) {
```

Comment:
Давай добавим возвращаемый тип метода
```suggestion
  public addPlayer(player: IPlayer): Promise<void> {
```

Reply:
✅